content/blog/2024-05-23-linebender-roadmap.md

```diff
@@ -0,0 +1,191 @@
+- **Issue:** TODO
+
+### Rewrite documentation
+
+- **Priority:** Medium.
+- **Assigned:** Olivier Faure.
+- **Description:** Masonry's documentation hasn't been updated since the port to Winit. Once other major changes are implemented, the documentation should be brought up to date. We should avoid procrastinating on documentation too much just because the architecture is still in progress.
+- **Issue:** TODO
+
+### Improve focus handling
+
+- **Priority:** Medium.
+- **Assigned:** Olivier Faure.
+- **Description:** We should improve tab focus in Masonry. Right now tab focus has some bugs, doesn't cover buttons, isn't updated by clicks, and uses architecture inherited from Druid that could be strongly simplified once other architectural changes are completed.
+- **Issue:** TODO
+
+### Improve pointer status handling
+
+- **Priority:** Medium.
+- **Assigned:** Olivier Faure.
+- **Description:** We should centralize and improve how Masonry handles pointers. In particular, we should add an explicit "pointer capture" API that makes other widgets not get events from that pointer until the pointer is released or lost.
+- **Issue:** [xilem#312](https://github.com/linebender/xilem/issues/312)
+
+
+## Parley
+
+### Create abstractions for text selection and editing
+
+- **Priority:** Highest.
+- **Assigned:** Chad Brokaw, Aaron Muir Hamilton.
+- **Description:** We want Parley to implement types and methods that Masonry and other editors will be able to use to create a text-editing widget, or to handle selection in non-editable text. The initial implementation may be located in Masonry.
+- **Issue:** [parley#52](https://github.com/linebender/parley/issues/52)
+
+### Inline block layout
+
+- **Priority:** High.
+- **Assigned:** Chad Brokaw.
+- **Description:** Parley should support laying out inline content the way web browsers do, where blocks are splices into your text and should be laid out according to available inline space the same way words are.
+- **Issue:** TODO
+
+### Write adequate documentation
+
+- **Priority:** High.
+- **Assigned:** Nico Burns.
+- **Description:** Parley currently has virtually no documentation. If we want it to become a fundational crate, we need to explain to people how to use it.
+- **Issue:** 
+
+### Software renderer
+
+- **Priority:** Low.
+- **Assigned:** Chad Brokaw.
+- **Description:** Create a software renderer for the glyphs created by Parley, so that users can easily import Parley in their library without depending on a graphics API like Vello. The software renderer should be black-and-white for simplicity. We'll call for volunteers to implement COLRv1 and other advanced features.
+- **Issue:** 
+
+
+## Xilem
+
+### Android bringup
+
+- **Priority:** Highest.
+- **Assigned:** Daniel McNab.
+- **Description:** We want Xilem to reliably work on Android. Running your Xilem app should be as simple as doing `cargo apk run` in your project.
```

Comment:
I'm not sure I want to be so specific about supporting exactly `cargo apk run`; there are concerns about how well it's possible to support Android with a non-gradle workflow. Additionally, we will almost certainly need at least some boilerplate in each app for Android support, which "in your project" *implies* wouldn't be needed.